applications/common/backend/src/main/resources/libs/kyanite/common/components/layouts/bannerwithbuttons/template/.content.json

```diff
@@ -0,0 +1,101 @@
+{
+  "jcr:primaryType": "nt:unstructured",
+  "renderAsHero": "false",
+  "sling:resourceType": "kyanite/common/components/section",
+  "container": {
+    "jcr:primaryType": "nt:unstructured",
+    "sling:resourceType": "kyanite/common/components/container",
+    "card": {
+      "jcr:primaryType": "nt:unstructured",
+      "type": "card",
+      "sling:resourceType": "kyanite/common/components/card",
+      "cardcontent": {
+        "jcr:primaryType": "nt:unstructured",
+        "sling:resourceType": "kyanite/common/components/card/cardcontent",
+        "title": {
+          "jcr:primaryType": "nt:unstructured",
+          "element": "h2",
+          "isSpaced": false,
+          "size": "is-2",
+          "text": "<p style=\"text-align: center\">Banner Section</p>",
```

Comment:
If using single quotes in the style definition - they would not have to be backslash-escaped. But if it works then it's OK

Reply:
This line is copied from a similar component, I didn't change the approach.